src/index.js

```diff
@@ -20,10 +20,69 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function render(info = {
+  left: 0,
+  operator: '+',
+  rigth: 0,
+  nextOperator: '+',
+}) {
```

Comment:
```suggestion
function render({
  left: 0,
  operator: '+',
  rigth: 0,
  nextOperator: '+',
}) {
```

이렇게 처리되면 객체로 받은걸 다시 info에 할당해주게 됩니다. (도루묵 느낌..?😅) 이렇게 처리하는게 좋겠군요!

- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#%ED%95%A8%EC%88%98_%EB%A7%A4%EA%B0%9C%EB%B3%80%EC%88%98%EB%A1%9C_%EC%A0%84%EB%8B%AC%EB%90%9C_%EA%B0%9D%EC%B2%B4%EC%97%90%EC%84%9C_%ED%95%84%EB%93%9C_%ED%95%B4%EC%B2%B4%ED%95%98%EA%B8%B0

Reply:
말씀해주신대로 반영했습니다!!👍